docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
       - database
     volumes:
       - ./app:/app
-    command: bash -c "../prepare_container.sh && python /app/manage.py collectstatic --no-input && python /app/manage.py runserver 0:5000"
```

Comment:
it's weird, some of them take both `--no-input` and `--noinput`, and others *only* take `--noinput`, so when we added collectstatic `--no-input` did nothing =P